index.html

```diff
@@ -11,68 +11,79 @@
       href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard.css"
     />
     <link rel="stylesheet" href="style.css" />
+    <meta property="og:image" content="/image/메타태그이미지.png" />
+    <meta property="og:title" content="판다마켓" />
+    <meta property="og:description" content="일상의 모든 물건을 거래해보세요" />
+    <meta property="og:url" content="https://www.codeit.kr" />
   </head>
   <body>
-    <header>
+    <header class="topBar">
       <a href="/">
-        <img src="/image/logo1.png" alt="main logo" />
+        <picture>
+          <source
+            srcset="/image/판다마켓.png"
+            media="all and (max-width:375px)"
+            alt="mobile log"
+          />
+          <img src="/image/logo1.png" alt="main logo" />
+        </picture>
       </a>
       <a class="login" href="/login.html">로그인</a>
     </header>
     <main>
       <div class="title">
         <h1>
-          일상의 모든 물건을<br />
+          일상의 모든 물건을<br class="break-banner" />
           거래해 보세요
         </h1>
-        <a href="/item.html">
-          <div class="button">구경하러 가기</div>
-        </a>
+        <div class="button">
+          <a href="/item.html"> 구경하러 가기 </a>
+        </div>
```

Comment:
```suggestion
          <a href="/item.html"> 구경하러 가기 </a>
```

이 부분은 `a` 태그만 있어도 괜찮겠네요 !

Reply:
<div class = "button">으로 구경하기버튼의 위치를 잡아주고 있는데
div를 없애게 되면 위치를 어떤식으로 지정해야 할까요?
div를 없애면 구경하기 버튼이 중간에 위치하게 됩니다!

a태그로만 위치를 잡아보려 해도 잘 잡히지가 않네요..ㅠ